k8s/init-services/02-config-map.yaml

```diff
@@ -0,0 +1,77 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ersms-forum-config
```

Comment:
Probably we will have to configure sth like this:
```yml
spring:
  config:
    activate:
      on-profile: kubernetes
  cloud:
    config:
      enabled: false
    kubernetes:
      reload:
        strategy: restart-context
        enabled: true
      config:
        enabled: true
        name: ersms-forum-config
```
to allow for centralized configuration